src/components/FlowRunCreateFormV2.vue

```diff
@@ -126,6 +127,10 @@
   const { errors, validate: validateParameters } = useSchemaValidation(schema, parameters)
 
   async function submit(): Promise<void> {
+    if (props.disabled) {
+      return
+    }
```

Comment:
I'm surprised this is needed with the submit button disabled? But also doesn't hurt.

Reply:
I tried a few different things and button smashing was still creating double runs sometimes. This I wasn't able to get past.

Reply:
I think just the order of execution and vue's rendering means the button can still be clicked multiple times before it actually gets disabled. This makes sure the submit never happens more than once.